### PR TITLE
fix(dashboard): unbroke mobile layout

### PR DESCRIPTION
**Pull request by mattnolting**
## Prerequisites

- [ ] Scope your styles to your individual card
- [ ] You are using translations when necessary with proper plural/singular modifications
- [ ] You are using the template card provided
- [ ] Use functional components & hooks (please)
- [ ] Attach screenshots (before and after)
- [ ] Make sure that all text in blue is populated with the correct link. If you're unsure what the url should be, just ask!

## What
Fixes tooltip in table wrapping. Unbroke layout.

### Before

Screen Shot 2021-04-22 at 4 36 06 PM

Screen Shot 2021-04-22 at 4 36 17 PM

Screen Shot 2021-04-22 at 4 36 23 PM

Screen Shot 2021-04-22 at 4 37 59 PM

Screen Shot 2021-04-22 at 4 37 27 PM

### After

Screen Shot 2021-04-22 at 4 39 24 PM

Screen Shot 2021-04-22 at 4 39 40 PM

Screen Shot 2021-04-22 at 4 40 07 PM

Screen Shot 2021-04-22 at 4 40 15 PM

Screen Shot 2021-04-22 at 4 40 22 PM

## Additional Info
[if needed]

## Code Review
@ryelo @AllenBW

## Design Review
@kybaker

### CODE REVIEW

**Comment by kybaker** [comment]
@mattnolting Looks excellent - ship it!

File: src/PresentationalComponents/Template/DataListItemTemplate.js [hunk 1]
```diff
@@ -63,47 +63,42 @@ export const DataListItemTemplate = ({
...
-                                        <Label color="red" icon={<DashboardIcon type='criticalSeverity' />}>
-                                            {capitalize(intl.formatMessage(messages.critical))}
-                                        </Label>
-                                    }
-                                </Flex>
-                            </div>
-                        </Flex>
+                        </div>
+                        <span className='ins-c-dashboard-data-list__date pf-u-color-200 pf-u-font-size-sm'>
+                            Newly released security rule &nbsp;<DateFormat type='onlyDate' date={dataListItemTemplateDate} />
```
Comment (AllenBW) [inline src/PresentationalComponents/Template/DataListItemTemplate.js:90]: Lets keep these strings translated

File: src/PresentationalComponents/Template/DataListItemTemplate.scss [hunk 2]
```diff
@@ -46,18 +57,18 @@
...
 
-    .ins-c-dashboard-data-list__title-row {
-        --pf-c-data-list__item-control--PaddingTop: var(--pf-global--spacer--sm);
-        --pf-c-data-list__item-control--PaddingBottom: var(--pf-global--spacer--sm);
-        --pf-c-data-list--m-compact__cell--PaddingTop: var(--pf-global--spacer--md);
-        --pf-c-data-list--m-compact__cell--PaddingBottom: var(--pf-global--spacer--md);
-        --pf-c-data-list__item-control--MarginLeft: 0;
-        --pf-c-data-list__item-control--MarginRight: calc(var(--pf-global--spacer--md) * -1);
-        --pf-c-data-list__toggle--MarginLeft: 0;
+    // .ins-c-dashboard-data-list__title-row {
```
Comment (AllenBW) [inline src/PresentationalComponents/Template/DataListItemTemplate.scss:60]: commented code needs killing

File: src/SmartComponents/NewRules/NewRules.scss [hunk 3]
```diff
@@ -21,5 +21,6 @@
 .ins-c-dashboard__list-item.pf-m-expanded {
   .ins-c-title-toggle {
     visibility: hidden;
+    display: none;
```
Comment (AllenBW) [inline src/SmartComponents/NewRules/NewRules.scss:24]: goooood catch

**Review by AllenBW — CHANGES_REQUESTED** [review]
@mattnolting ah few feedback items, also if yer gonna change the info tooltip in two places, oughta change it in the third so everything looks the same (advisor info tooltip is still a button)

alsoooo after taking a look, just wanted to confirm, this work is focused on making cards more compact when displayed in smaller viewports?

**Comment by AllenBW** [comment]
@mattnolting still some linting to work through

**Review by AllenBW — APPROVED** [review]
👍 will push to prod beta immediately, prod-stable this eve

**Pull request merged**